src/uPlot.js

```diff
@@ -1761,21 +1816,38 @@ export default function uPlot(opts, data, then) {
 		}
 	}
 
-	function mouseUp(e, src, _x, _y, _w, _h, _i) {
-		if ((e == null || filtMouse(e))) {
+	function mouseUp(e, src, _x, _y, _w, _h, _i, _drag) {
```

Comment:
in what situations would _drag during an isSyncReq be different from one asserted from a difference in local e.g. mouseLeft1/mouseLeft0 positions? since mousedown/mousemove/cachemouse is done across all synced charts, wouldnt each chart have local knowledge of whether a drag happend by simply checking the `dragging` boolean?

Reply:
when you are checking if drag is done in mouseup instead of in mousemove; no longer necessary

Reply:
actually one of these is necessary to keep cursor locks working, not sure which. Investigating....

Reply:
This is not the one, see comment in other review